Update README.md for kubernetes_cluster_autoscaler (#20061)
DataDog/integrations-core: kubernetes_cluster_autoscaler/README.md

**Note**: The listed metrics can only be collected if they are available. Some metrics are generated only when certain actions are performed. 

The only parameters required for configuring the `kubernetes_cluster_autoscaler` check are:

* CONTAINER_NAME

>   Describe the pod belonging to your cluster autoscaler, and find the container name.

Reviewer: ```suggestion
  Name of the container of the cluster autoscaler controller.
```

metadata:
  name: '<POD_NAME>'
  annotations:

>     ad.datadoghq.com/<CONTAINER_NAME>.checks: |

Reviewer: This `<CONTAINER_NAME>` has to match the name of the container defined below line 76.
If we keep a container name hard-coded to `controller` line 76, it should be fine to keep `controller` here as well.
If we want to be able to customize the name of the container, then, we should also put `<CONTAINER_NAME>` below, line 76.

Author: I'll adjust line 76 to match, because the container name is set by the cluster autoscaler deployment, and could be anything.
